apps/fhir/bluebutton/utils.py

```diff
@@ -909,3 +912,25 @@ def build_oauth_resource(request, format_type="json"):
         ]
 
     return security
+
+
+def get_page_size(request):
```

Comment:
move these functions into a file called `pagination`

Reply:
pagination.py created.
get_page_size(request.GET)   # receives QueryDict
copes with alphas in parameter (returns -1)
handles _count first
Returns value or DEFAULT_PAGE_SIZE
returns value when first value found so _count takes precedence over count.
tests written to cope with receiving both parameters and for receiving alpha values

is_int function tests value can be converted to int otherwise I was getting an unwrapped ValueError in the function.